ocaml/idl/datamodel.ml

```diff
@@ -5598,6 +5598,8 @@ let all_relations =
     (_network_sriov, "logical_PIF"), (_pif, "sriov_logical_PIF_of");
 
     (_certificate, "host"), (_host, "certificates");
+    (_certificate, "pool"), (_pool, "certificates");
+    (_certificate, "validated_by"), (_certificate, "validates");
```

Comment:
I was worried that this would create a loop, but we already have similar situations with VM snapshots/snapshot_of, so this should be fine.

Reply:
In the documentation it's specified that setting the `validated_by` field will automatically update the corresponding `validates` field in the referenced object with the correct reference.